### PR TITLE
python310Packages.omegaconf: Enable more tests

### PR DESCRIPTION
**Pull request by onny**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/pydevd/default.nix [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    pytestCheckHook
+    trio
+    untangle
+  ];
+
+  pythonImportsCheck = [ "pydevd" ];
+
+  disabledTests = [
+    # Require network connection
+    # Missing import module
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/pydevd/default.nix:33]: Whats up with this?

File: pkgs/development/python-modules/pydevd/default.nix [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+, numpy }:
+
+buildPythonPackage rec {
+  pname = "pydevd";
+  version = "2.8.0";
+
+  src = fetchFromGitHub {
+    owner = "fabioz";
+    repo = "PyDev.Debugger";
+    rev = "ac14df98d09c43cd02e5546165a099218fbfbe94";
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/pydevd/default.nix:17]: there are tags https://github.com/fabioz/PyDev.Debugger/releases/tag/pydev_debugger_2_8_0

**Comment by onny** [comment]
Thank you @SuperSandro2000 for reviewing :)

File: pkgs/development/python-modules/pydevd/default.nix [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+}:
+
+buildPythonPackage rec {
+  pname = "pydevd";
+  version = "2.8.0";
+
+  src = fetchFromGitHub {
+    owner = "fabioz";
+    repo = "PyDev.Debugger";
+    rev = "pydev_debugger_2_8_0";
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/pydevd/default.nix:18]:
```suggestion
    rev = "pydev_debugger_${lib.replaceStrings ["."] ["_"] version}";
```

File: pkgs/development/python-modules/pydevd/default.nix [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    "test_completion_sockets_and_messages"
+    "test_path_translation"
+    "test_attach_to_pid_no_threads"
+    "test_attach_to_pid_halted"
+    "test_remote_debugger_threads"
+    "test_path_translation_and_source_reference"
+    "test_attach_to_pid"
+    "test_terminate"
+    "test_gui_event_loop_custom"
+    # AssertionError: assert '/usr/bin/' == '/usr/bin'
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/pydevd/default.nix:41]: Did we maybe forgot to patch some path?

Comment (onny) [reply pkgs/development/python-modules/pydevd/default.nix:41]: Hm I'm really not sure about this. I created a upstream bug report and added a comment here.

**Pull request merged**